tmt/utils.py

```diff
@@ -190,8 +190,8 @@ def opt(self, option, default=None):
         elif option in ['debug', 'verbose']:
             winner = local if local else parent
             # Convert tuple of booleans into debug/verbose level (int)
-            if isinstance(winner, tuple):
-                winner = len(winner)
+            if isinstance(winner, int):
```

Comment:
Isn't this condition completely redundant now? I think it will always be an integer since you specify `count=True` and `default=0`. And thanks to the default, the None check below shouldn't be needed as well, I'd assume.

I was also wondering whether the default couldn't cause problems in the line `winner = local if local else parent` which used to check if `local` is defined but since there is no way how to explicitly use 0 for the counting option, value 0 means that it isn't defined so that should be fine.

One more note, looks like some commits from the master branch got into this PR, rebasing to master should hopefully fix this.

Reply:
Agree with @FrNecas, the condition now seems to be redundant. After the force push I still see the unrelated commits in the pull request. Please, do a rebase or a cherrypick.

Reply:
About:
winner = local if local else parent
should be completely removed?
 But what about inheritance of value from parent?
 What about explicit check for local == 0 or similar?

Reply:
No, I think that should be kept (sorry for the confusion, was just brainstorming if that works fine after the changes and it should in my opinion ;) ).